### PR TITLE
fix(reverse): fix lookup problems with url with no edit mode markers

### LINKED ISSUE #274: lookup gives 404 for correct document
**Description**
https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v2?owner=adobe&repo=theblog&ref=master&path=%2F&lookup=https%3A%2F%2Fadobe.sharepoint.com%2Fsites%2FTheBlog%2F_layouts%2F15%2FDoc.aspx%3Fsourcedoc%3D%257BBFD9A19C-4A68-4DBF-8641-DA2F1283C895%257D%26file%3Dindex.docx%26action%3Ddefault%26mobileredirect%3Dtrue%26cid%3Db6e62d68-f53a-471f-a686-90e69458fdd7

### PR DESCRIPTION
**Pull request by tripodsan**
fixes #274

### CODE REVIEW

File: src/onedrive-reverse.js [hunk 1]
```diff
@@ -58,7 +58,9 @@ async function reverseLookup(opts) {
     log,
   });
 
-  const segs = uri.pathname.substring(1).split('/');
+  const segs = uri.pathname.substring(1)
+    .split('/')
+    .filter((s) => s !== 'r' && s !== ':w:');
```
Comment (rofe) [inline src/onedrive-reverse.js:63]: Do we need special handling for Excel (`:x:`) too?

**Review by rofe — APPROVED** [review]

File: src/onedrive-reverse.js (hunk 1 above)
Comment (tripodsan) [reply src/onedrive-reverse.js:63]: maybe ? ... I'll add it

Comment (tripodsan) [inline src/onedrive-reverse.js:63]:
```suggestion
    .filter((s) => s !== 'r' && s !== ':w:' && s !== ':x:');
```

**Pull request merged**